components/automate-ui/src/app/entities/api-tokens/api-token.reducer.ts

```diff
@@ -61,7 +66,13 @@ export function apiTokenEntityReducer(
     }
 
     case ApiTokenActionTypes.GET: {
-      return set(GET_STATUS, EntityStatus.loading, state) as ApiTokenEntityState;
+      return set(
+        GET_STATUS,
+        EntityStatus.loading,
+        // clear token state to prevent data leakage
```

Comment:
for this comment and the one above, I'm looking for something around authorization leakage-- in a world without the auth concerns, this wouldn't matter as much. That said, I'm not totally sure on the wording 🤔 
> leaking potentially unauthorized data 
? 🤷‍♀ . Non-blocking.